README.md

```diff
@@ -66,6 +66,7 @@ The config file is a yaml formatted file with the following fields:
 - `nickserv_identify`, optional, on connect this message will be sent: `PRIVMSG nickserv IDENTIFY <value>`, you can provide both a username and password if your ircd supports it
 - `cooldown_duration`, optional, default 86400 (24 hours), time in seconds for a discord user to be offline before it's puppet disconnects from irc
 - `show_joinquit`, optional, default true, displays JOIN, PART, QUIT, KICK on discord
+- `max_nick_length`, optional, default 30, Maximum allowed nick length
```

Comment:
Please can you add this to config.yml and the YAML snippet in this readme file

Reply:
I've updated the config.yml. 
It seems like having a snippet and an example config is likely to lead to drifting values.
So I added a Matrix of all the config values with defaults, restarts and optionality to try to reduce the places that need to be updated